tcms/xmlrpc/forms.py

```diff
@@ -30,11 +32,32 @@ class UpdateCaseForm(XMLRPCUpdateCaseForm):
                                          widget=XMLRPCCheckboxInput)
 
 
-class NewPlanForm(XMLRPCNewPlanForm):
+class NewPlanForm(NewPlanForm):
+    text = CharField()
+
     is_active = BooleanField(label="Active", required=False,
                              widget=XMLRPCCheckboxInput)
 
 
-class EditPlanForm(XMLRPCEditPlanForm):
+class EditPlanForm(NewPlanForm):
```

Comment:
Hmmm, this not doing what you think. `NewPlanForm` as a name will be redefined above and this class will inherit not from the base class but from the one in the same module. I'm kind of surprised there's no name collision going on above.

Reply:
This was just an intermediate operation. It is not so in the final version. Anyhow I have fixed it in the commit, and not everyone of the commits should be working by itself (this was not the case before)